Tests.Integration.Presentation.Web/ItSystem/CatalogTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Integration.Presentation.Web.Tools.Model;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.ItSystem
+{
+    public class CatalogTests : WithAutoFixture
+    {
+        private readonly KitosCredentials _apiUser;
+
+        public CatalogTests()
+        {
+            _apiUser = TestEnvironment.GetApiUser();
+        }
+
+        [Fact]
+        public async Task Api_User_Can_Get_IT_System_Data_From_Specific_System_From_own_Organization()
+        {
+            //Arrange
+            var loginDto = ObjectCreateHelper.MakeSimpleLoginDto(_apiUser.Username, _apiUser.Password);
+            var token = await HttpApi.GetTokenAsync(loginDto);
+
+            //Act
+            using (var httpResponse = await HttpApi.GetWithTokenAsync(TestEnvironment.CreateUrl("odata/ItSystems(1)"), token.Token))
+            {
+                var response = httpResponse.ReadResponseBodyAs<Core.DomainModel.ItSystem.ItSystem>();
+                //Assert
+                Assert.Equal(HttpStatusCode.OK, httpResponse.StatusCode);
+                Assert.Equal("DefaultTestItSystem", response.Result.Name);
+            }
+        }
+
+        [Fact]
+        public async Task Api_User_Can_Get_All_IT_Systems_Data_From_Own_Organizations()
+        {
+            //Arrange
+            var loginDto = ObjectCreateHelper.MakeSimpleLoginDto(_apiUser.Username, _apiUser.Password);
+            var token = await HttpApi.GetTokenAsync(loginDto);
+
+            //Act
+            using (var httpResponse = await HttpApi.GetWithTokenAsync(TestEnvironment.CreateUrl("odata/ItSystems"), token.Token))
+            {
+                var response = httpResponse.ReadListResponseBodyAs<Core.DomainModel.ItSystem.ItSystem>();
+                //Assert
+                Assert.Equal(HttpStatusCode.OK, httpResponse.StatusCode);
+                Assert.Equal("DefaultTestItSystem", response.Result.First().Name);
```

Comment:
Det er en ret farlig antagelse. Ret til en "Any" med en condition på det navn. Der er ingen garanti for at DefaultItSystem står først, når først vi kommer i gang med flere tests

Reply:
Check blot at name ikke er null